plugins/promdump.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: promdump
+spec:
+  version: v0.2.2
+  homepage: https://github.com/ihcsim/promdump
+  shortDescription: promdump dumps the head and persistent blocks of Prometheus.
+  description: |
+    promdump dumps the head and persistent blocks of Prometheus. It supports
+    filtering the persistent blocks by time range.
+
+    promdump is a tool that can be used to dump Prometheus data blocks. It is
+    different from the 'promtool tsdb dump' command in such a way that its output
+    can be re-used in another Prometheus instance. And unlike the Promethues TSDB
+    'snapshot' API, promdump doesn't require Prometheus to be started with the
```

Comment:
```suggestion
    'snapshot' command, promdump doesn't require Prometheus to be started with the
```

Reply:
There is an actual [`snapshot` HTTP endpoint](https://prometheus.io/docs/prometheus/latest/querying/api/#snapshot).